recipe/meta.yaml

```diff
@@ -239,6 +243,45 @@ outputs:
       imports:
         - ray.tune
 
+  - name: ray-train
+    # exactly equivalent to ray-tune
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('ray-default', exact=True) }}
+        - pandas
+        - requests
+        - tabulate
+        - tensorboardX >=1.9
+    test:
+      imports:
+        - ray.train
+
+  - name: ray-air
```

Comment:
The outputs are alphabetized, though I guess `ray-all` should stay first as the collection of everything. Speaking of `ray-all`, new outputs need to be added there as well